### ISSUE TITLE
Activate "View  in full context" on more than just the "ts" field

### ISSUE THREAD

**Issue by philrz** [brimsec/brim#923]
As a user, the first time I tried to use the  "View in full context" right-click option (https://github.com/brimsec/brim/pull/894) I was briefly confused to find it was "grayed out" when I right-clicked on an event. Because I knew @r-franke approached this feature after using the "Use as start/end time" options (which arguably make sense to tie to a `time`-type field like `ts`) I thought maybe it was similarly tied to `ts`. Indeed it was, so once I knew that, it worked as expected. However, I suspect new users might not make this connection, since because of the way it's worded it seems like it may be tied to the whole of the event (i.e. right-click on any field) rather than just one particular field.

(I discussed this on Slack with @r-franke who authored https://github.com/brimsec/brim/pull/894 and he seemed to agree with my assessment.)

**Comment by r-franke** [comment]
Yeah we briefly touched the working of "Use as start/end time" as well during that discussion however we didn't finish it. So I'll raise it here, curious what others are thinking.

Especially now that we remember the scrollposition it may be more often then not the case that users don't have their scrollbars all the way to the left when they're examining a certain column. If I'm 3/4 quarters to the right and I spot something of interest then I should be able to click "Use as start/end time"  and "View in full context". Otherwise I have to mark the line I'm interested in, scroll to the left, right-click, click. Seems unnecessary.

**Comment by jameskerr** [comment]
I'm in favor of having both "Use as start/end time" and "View in full context" available for all the cells in a row. I think the wording should change to: "Use ts as start time" and "Use ts as end time".

**Comment by philrz** [comment]
Another community user recently expressed a desire for this functionality and due to its difficult-to-find nature they had no idea it was already there to use. I'll mark this for consideration for the next milestone in the event this might be something quick we can knock out.
